src/Fantomas.Tests/SynConstTests.fs

```diff
@@ -301,13 +301,13 @@ let ``hex escape in string literal should be preserved, 1508`` () =
     formatSourceString
         false
         """let hexEscape = "\x00"
-let controlEscapes = "\a \b \f \v"
+let controlEscapes = "\a \b \f \t \v"
 """
         config
     |> should
         equal
         """let hexEscape = "\x00"
-let controlEscapes = "\a \b \f \v"
+let controlEscapes = "\a \b \f \t \v"
```

Comment:
Could you please write a new test based on your original report?
The other characters in the string already make the string trivia kick in.
So this test would indeed pass without updating the regex.

Reply:
Done